assets/components/service-ca/deployment.yaml

```diff
@@ -23,45 +22,45 @@ spec:
         app: service-ca
         service-ca: "true"
     spec:
-      serviceAccount: service-ca
       serviceAccountName: service-ca
       containers:
-      - name: service-ca-controller
-        image: {{ .ReleaseImage.service_ca_operator }}
-        imagePullPolicy: IfNotPresent
-        command: ["service-ca-operator", "controller"]
-        ports:
-        - containerPort: 8443
-        securityContext:
-          runAsNonRoot: true
-        resources:
-          requests:
-            memory: 120Mi
-            cpu: 10m
-        volumeMounts:
-        - mountPath: /var/run/secrets/signing-key
-          name: signing-key
-        - mountPath: /var/run/configmaps/signing-cabundle
-          name: signing-cabundle
+        - name: service-ca-controller
+          image: {{ .ReleaseImage.service_ca_operator }}
+          imagePullPolicy: IfNotPresent
+          command: ["service-ca-operator", "controller"]
+          ports:
+            - containerPort: 8443
+          securityContext:
+            runAsNonRoot: true
+          resources:
+            requests:
+              memory: 120Mi
+              cpu: 10m
+          volumeMounts:
+            - mountPath: /var/run/secrets/signing-key
+              name: signing-key
+            - mountPath: /var/run/configmaps/signing-cabundle
+              name: signing-cabundle
       volumes:
-      - name: signing-key
-        secret:
-          secretName: {{.TLSSecret}}
-      - name: signing-cabundle
-        configMap:
-          name: {{.CAConfigMap}}
-      # nodeSelector:
-      #   node-role.kubernetes.io/master: ""
+        - name: signing-key
+          secret:
+            secretName: {{.TLSSecret}}
+        - name: signing-cabundle
+          configMap:
+            name: {{.CAConfigMap}}
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
```

Comment:
It's unlikely that this was commented out in our code, you may want to figure out why this was here.

Reply:
We originally didn't have node roles set; this was fixed in #869 only recently; we probably missed removing the comment.